src/ReactiveUI.SourceGenerator.Tests/REACTIVE/ReactiveGeneratorTests.FromReactiveProperiesWithAttributes#TestNs.TestVM.Properties.g.verified.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace TestNs
 {
-    /// <summary>
```

Comment:
I'm guessing the offset is just caused by a `"""` raw string literal, where the first line is not aligned with the ending quotes.